wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+EmptyState.swift

```diff
@@ -55,6 +55,9 @@ extension ConversationListViewController {
                 subheadline: Strings.Oneonone.subheadline(domain).attributedString,
                 showArrow: !isIPadRegular(),
                 showButton: isIPadRegular())
+        case .folder:
+            // FIXME: [WPB-13905] Disallow this state
```

Comment:
We shouldn't really show an empty state in the folders case. Instead if there are no folders or no conversations in the folder we should remove the filter. This will be fixed in `WPB-13905`

Reply:
Please also consider the edge case that conversation list for any given folder is currently presented and on another client you remove the last conversation from the folder.

Reply:
I guess that is what I intended for `WPB-13905`. I will try to clarify the ticket further